dth/client.go

```diff
@@ -281,7 +281,7 @@ func (c *S3Client) listObjectFn(ctx context.Context, continuationToken, prefix,
 		input = &s3.ListObjectsV2Input{
 			Bucket:       &c.bucket,
 			Prefix:       prefix,
-			MaxKeys:      maxKeys,
+			MaxKeys: 	  aws.Int32(maxKeys),
```

Comment:
is this issue documented on public aws sdk?

Reply:
Here is the CR that resulted in changing this behavior - https://github.com/aws/aws-sdk-go-v2/commit/4bd06b93f8627928c17604383bd8639f2cb23739